.gitignore

```diff
@@ -6,3 +8,5 @@
 /pkg/
 /spec/reports/
 /tmp/
+/test/
```

Comment:
This shouldn't be part of the `.gitignore`
```suggestion
```

Reply:
Oh duh right. I put in the correct paths.